src/redux/migrations.ts

```diff
@@ -8,7 +8,27 @@ import { SendingFiatAccountStatus } from 'src/fiatconnect/slice'
 import { REMOTE_CONFIG_VALUES_DEFAULTS } from 'src/firebase/remoteConfigValuesDefaults'
 import { AddressToDisplayNameType } from 'src/identity/reducer'
 import { TokenTransaction } from 'src/transactions/types'
-import { Currency } from 'src/utils/currencies'
+import { CiCoCurrency, Currency } from 'src/utils/currencies'
+
+export function updateCachedQuoteParams(cachedQuoteParams: {
+  [providerId: string]: {
+    [kycSchema: string]: any
+  }
+}) {
+  const newCachedQuoteParams: any = {}
```

Comment:
why not use a more descriptive return type here? seems like it would only make the code safer